sys/linux/test/landlock_ptrace

```diff
@@ -0,0 +1,32 @@
+# Creates independent Landlock hierarchies.
+#
+# fork() is not available for the following architectures. Set "manual" to
+# avoid test errors from csource.
+# requires: manual threaded=0 -arch=arm64 -arch=riscv64
```

Comment:
I did put "manual" as the first constraint though.

Reply:
The issue comes from TestParsing, not TestSysTests.

Reply:
TestSysTests indeed takes the "manual" into account.

Reply:
You are right.
https://github.com/google/syzkaller/commit/2bd9619f762176527aaf28fb26e4a08b614b55df should make `arch` requirements work properly.